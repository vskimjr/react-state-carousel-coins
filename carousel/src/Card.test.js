import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

test("renders without crashing", function () {
  render(<Card />);
});

test("matches snapshot", function () {
  const { container } = render(<Card />);
  expect(container).toMatchSnapshot();
});