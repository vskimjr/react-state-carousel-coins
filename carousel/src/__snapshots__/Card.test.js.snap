// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image 
       of 
      .
    </small>
  </div>
</div>
`;
